blog/content/post/2019-07-03-how-to-safely-and-easily-update-your-firmware.md

```diff
@@ -0,0 +1,278 @@
+what they think about joining this project. Then there is a possibility to
+request an account on the LVFS site, get legal permission to redistribute the
+firmware and start uploading new updates.
+
+Single upload contains binary file and at least one XML file with required
+metainfo about target device and firmware, all packed as a cabinet archive,
+which is consistent with the [Microsoft Update][req] requirements.
+
+![cabinet archive example](/img/lvfs-cabinet-archive.png)
+
+Archives specially prepared by authorized vendors are the key to the whole
+update process. Metadata from the LVFS brings the details about available
+updates to the user together with patch notes description. The XML file needs to
+meet the structure and content standards which can be found in the LVFS examples.
+
+Example of v4.9.0.1 coreboot release for Libretrend LT1000:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 3mdeb <contact@3mdeb.com> -->
+<component type="firmware">
+  <id>com.Libretrend.LT1000.firmware</id>
+  <name>LT1000 Coreboot</name>
+  <summary>Firmware for the Libretrend LT1000 platform</summary>
+  <description>
+    <p>
+      The platform can be updated using flashrom (internal programmer).
+    </p>
+  </description>
+  <provides>
+    <!-- this is a suitable HWID, found using `fwupdmgr hwids` -->
+    <firmware type="flashed">52b68c34-6b31-5ecc-8a5c-de37e666ccd5</firmware>
+  </provides>
+  <url type="homepage">http://www.3mdeb.com/</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Proprietary</project_license>
+  <developer_name>Libretrend</developer_name>
+  <releases>
+    <release urgency="high" version="4.9.0.1" date="2019-01-26" install_duration="60">
+      <checksum filename="com.Libretrend.LT1000.firmware.bin" target="content"/>
+      <description>
+        <p>This release features:</p>
+        <ul>
+          <li>Moved console initialization and sign-of-life to bootblock</li>
+          <li>Fixed the CBFS size to 6MB</li>
+          <li>Minor build fixes</li>
+          <li>Rebased on official coreboot repository commit 2ef569a4</li>
+        </ul>
+      </description>
+    </release>
+  </releases>
+
+  <custom>
+    <value key="LVFS::VersionFormat">quad</value>
+    <value key="LVFS::UpdateProtocol">org.flashrom</value>
+  </custom>
+
+  <categories>
+    <category>X-Device</category>
+  </categories>
+
+  <!-- only newer versions of fwupd know how to write to this hardware -->
+  <requires>
+    <id compare="ge" version="1.1.2">org.freedesktop.fwupd</id>
+  </requires>
+</component>
+```
+
+We can announce, that from the beginning of 2019 we are approved ODM for
+Libretrend company and our status can be tracked on LVFS [vendor list][vl].
+Until now we have uploaded 4 custom firmware versions: `v4.8.0.2`, `v4.8.0.3`,
+`v4.8.0.4` and the newest `v4.9.0.1` specially refined coreboot for Librebox
+platforms (LT1000).
```

Comment:
@artur-rs in blog post description that will go to socia-media we should have Libretrend and that we are LVFS approved  ODM mentioned.